bitstore/services.py

```diff
@@ -20,59 +21,12 @@ def public_key():
     return _public_key
 
 
-def verify(auth_token, owner):
+def verify(auth_token, owner=None):
     """Verify Auth Token.
     :param auth_token: Authentication token to verify
     :param owner: dataset owner
     """
-    if not auth_token:
-        return False
+    verifyer = Verifyer(public_key=public_key())
```

Comment:
1. no need to create an object each time.
2. pass the auth endpoint and let Verifyer get the public key from there (and not using the public_key method)

Reply:
@akariv thinking to refactor this as done in specstore (pass it from blueprint) WHYT? 
* https://github.com/datahq/specstore/blob/master/flowmanager/controllers.py#L81-L84
* https://github.com/datahq/specstore/blob/master/flowmanager/blueprint.py#L17-L30

Reply:
Yes, looks good.